vividus-plugin-applitools/src/main/java/org/vividus/visual/eyes/model/ApplitoolsVisualCheck.java

```diff
@@ -16,30 +16,21 @@
 
 package org.vividus.visual.eyes.model;
 
-import java.net.URI;
 import java.util.Set;
 
-import com.applitools.eyes.MatchLevel;
+import com.applitools.eyes.config.Configuration;
 
-import org.openqa.selenium.Dimension;
 import org.vividus.ui.action.search.Locator;
 import org.vividus.visual.model.AbstractVisualCheck;
 import org.vividus.visual.model.VisualActionType;
 
 public class ApplitoolsVisualCheck extends AbstractVisualCheck
 {
-    private String executeApiKey;
     private String readApiKey;
-    private String hostApp;
-    private String hostOS;
-    private Dimension viewportSize;
-    private MatchLevel matchLevel;
-    private URI serverUri;
-    private String appName;
     private String batchName;
-    private String baselineEnvName;
     private Set<Locator> elementsToIgnore = Set.of();
     private Set<Locator> areasToIgnore = Set.of();
+    private Configuration configuration;
```

Comment:
```suggestion
    private String batchName;
    private Configuration configuration;
    private String readApiKey;
    private Set<Locator> elementsToIgnore = Set.of();
    private Set<Locator> areasToIgnore = Set.of();
```

and I guess default values for "ignores" are not needed anymore, right?